snuba/admin/notifications/slack/client.py

```diff
@@ -49,3 +57,6 @@ def post_message(
 
         if not is_ok:
             logger.error(f"Slack error: {str(error_option)}")
+
+
+slack_client = SlackClient()
```

Comment:
I think you can remove this. Just instantiate it in the scenarios where you need it, rather than on load of this module. I see that you are calling `client = SlackClient()` in some other places anyway and not even using this instance.

Reply:
I can change the audit log to instantiate it, but im not sure why it would be a problem to have one instance of the slack client if we wanted to use it more places that the audit log